Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1422,6 +1422,122 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.21,
 )
 
+class UpgradeWorkflow_ProdLikeRunningPU(UpgradeWorkflow_ProdLike):
+    def __init__(self, suffix, offset, fixedPU,
+         steps = [],
+        PU = [
+            'GenSimHLBeamSpot14',
+            'Digi',
+            'DigiTrigger',
+            'Reco',
+            'RecoGlobal',
+            'RecoNano',
+            'HARVEST',
+            'HARVESTGlobal',
+            'HARVESTNano',
+            'MiniAOD',
+            'ALCA',
+            'ALCAPhase2',
+            'Nano',
+        ]):
+        super(UpgradeWorkflow_ProdLikeRunningPU, self).__init__(steps, PU, suffix, offset)
+        self.__fixedPU = fixedPU
+    def setupPU_(self, step, stepName, stepDict, k, properties):
+        #  change PU skipping ALCA and HARVEST
+        if stepDict[stepName][k] is not None and '--pileup' in stepDict[stepName][k]:
+            stepDict[stepName][k]['--pileup'] = 'AVE_' + str(self.__fixedPU) + '_BX_25ns'
+    def condition(self, fragment, stepList, key, hasHarvest):
+        # lower PUs for Run3
+        return (fragment=="TTbar_14TeV") and (('2026' in key) or ('2021' in key and self.__fixedPU<=100))
+
+# The numbering below is following the 0.21 for ProdLike wfs
+# 0.21N would have been a more natural choice but the
+# trailing zeros are ignored. Thus 0.21N1 is used
+
+upgradeWFs['ProdLikePU10'] = UpgradeWorkflow_ProdLikeRunningPU(
+    suffix = '_ProdLikePU10',
+    offset = 0.21101,
+    fixedPU = 10,
+)
+
+upgradeWFs['ProdLikePU20'] = UpgradeWorkflow_ProdLikeRunningPU(
+    suffix = '_ProdLikePU10',
```

Comment:
Seem to be a typo here, sorry I just see it.

Reply:
Right! Fixing it, thanks